README.md

```diff
@@ -15,4 +15,4 @@ AI supports next complexity levels: Random move, Win strategy
 
 Actions: New, Save, Load
 
-The proposal for data driven approach: configuration (saved in configuration file or DB): board size, distance, tool size, win statement, lose statement, AI parameters
```

Comment:
1) wrong repo name
2) please revert all changes and create correct pull request
3) tag is wrong

Reply:
Edited repo name and git tag